SimpleGame/src/main/scala-2.11/edu/gemini/experimental/jluhrs/EventStream.scala

```diff
@@ -0,0 +1,26 @@
+package edu.gemini.experimental.jluhrs
+
+/**
+  * Created by jluhrs on 6/2/16.
+  */
+
+// My attempt to describe external inputs as a stream of values with a timestamp.
+
+object EventStream {
+  import Timer._
+
+  type Event[A] = (Time, A)
+  type EventStream[A] = Stream[Option[Event[A]]]
```

Comment:
You are right. I'm not sure what I was thinking. It was about how to deal with an empty input stream, but I should have just used Stream.Empty.